sysl2/sysl/grammar/lexer_impl.go

```diff
@@ -116,7 +118,7 @@ func GetNextToken(l *SyslLexer) antlr.Token {
 	}
 
 	next := l.BaseLexer.NextToken()
-	if os.Getenv("SYSL_LEXER_LOG") != "" {
+	if syslLexerLog {
```

Comment:
lol, it was getting the env var once every time something called `GetNextToken` was called. i bet that was a lot of lookups...